loader/include/Geode/utils/web.hpp

```diff
@@ -122,12 +134,20 @@ namespace geode::utils::web {
 
         WebRequest& userAgent(std::string_view name);
 
+        WebRequest& encoding(std::string_view encoding);
+
         WebRequest& timeout(std::chrono::seconds time);
 
+        WebRequest& range(std::pair<std::uint64_t, std::uint64_t> byteRange);
+
         WebRequest& certVerification(bool enabled);
+        WebRequest& transferBody(bool enabled);
+        WebRequest& followRequest(bool enabled);

```

Comment:
could this be renamed to `followRedirects` or is that a different thing

Reply:
there is no followRedirects lol, it used to be default enabled, this PR made it a default false option. I can rename it to `followRedirects` if you want

Reply:
it now defaults to not following redirects? i feel like that is a bad idea, subtle api change :nedfast:

Reply:
? https://github.com/SpaghettDev/geode/blob/b8509107a971c31a47babe337c8cb83ad29e8083/loader/src/utils/web.cpp#L202

Reply:
yes honey, it used to default to true and now, as you said, it defaults to false

Reply:
mb that was a mistake on my part, i meant i made it a default-true option